app/service/rest.py

```diff
@@ -312,6 +318,29 @@ def update_whitelist(service_id):
         return '', 204
 
 
+@service_blueprint.route('/<uuid:service_id>/deactivate', methods=['POST'])
+def deactivate_service(service_id):
+    service = dao_fetch_service_by_id(service_id)
+
```

Comment:
Typically the endpoints are only updating one resource at a time. The @transactional decorator is useful, but make it difficult to compose updates. We took the same approach when updating the resources for the notification stats (when it was more than one table)